src/lib.js

```diff
@@ -27,6 +27,8 @@ var curlConverter = {
             .option('-G, --get', 'Forces the request to be sent as GET, with the --data parameters appended to the query string', null)
             .option('-H, --header [string]', 'Add a header (can be used multiple times)', collectValues, [])
             .option('-X, --request [string]', 'Specify a custom request mehod to be used', null)
+            .option('-I, --head', 'Forces the request to be sent as HEAD, with the --data parameters appended to the query string', null)
+            .option('-T, --upload-file', 'Forces the request to be sent as PUT', null)
```

Comment:
Not accurate. `--upload-file` takes an argument for the file to upload. Forcing the request to be sent as PUT is a side-effect.